### PR TITLE
Enable attachments on all networks but Mainnet

### PR DESCRIPTION
**Pull request by turbolent**
⚠️ Depends on #4017

### CODE REVIEW

**Review by janezpodhostnik — APPROVED** [review]

**Review by SupunS — APPROVED** [review]

**Review by pattyshack — APPROVED** [review]

**Comment by turbolent** [comment]
bors merge

**Comment by turbolent** [comment]
bors merge

**Comment by janezpodhostnik** [comment]
this test seems to be failing 
```
=== RUN   TestAuthAccountCapabilities/account_linking_disallowed
    fvm_test.go:2089: 
        	Error Trace:	/home/runner/work/flow-go/flow-go/fvm/fvm_test.go:2089
        	            				/home/runner/work/flow-go/flow-go/fvm/fvm_test.go:93
        	            				/home/runner/work/flow-go/flow-go/fvm/fvm_test.go:2092
        	            				/home/runner/work/flow-go/flow-go/fvm/fvm_test.go:2100
        	Error:      	Error "[Error Code: 1101] error caused by: 1 error occurred:\n\t* transaction execute failed: [Error Code: 1101] cadence runtime error: Execution failed:\nerror: account linking is not allowed\n --> 36d99853444926b28da9aac00e5646e07baa8c2f0571c56b3ac460835490cf95:6:16\n  |\n6 | \t\t\t\t\t\t          acct.linkAccount(/private/foo)\n  | \t\t\t\t\t\t          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\n\n" does not contain "no member `linkAccount`"
        	Test:       	TestAuthAccountCapabilities/account_linking_disallowed
```

File: fvm/fvm_test.go [hunk 1]
```diff
@@ -2086,6 +2086,7 @@ func TestAuthAccountCapabilities(t *testing.T) {
 						require.NoError(t, tx.Err)
 					} else {
 						require.Error(t, tx.Err)
+						require.ErrorContains(t, tx.Err, "no member `linkAccount`")
```
Comment (turbolent) [inline fvm/fvm_test.go:2089]: The function is now always statically available

```suggestion
```

**Comment by turbolent** [comment]
@janezpodhostnik Thanks for having a look! I should have not added an unrelated assertion, which is now outdated. I removed it, which should hopefully fix the build/tests

**Comment by turbolent** [comment]
bors merge

**Pull request merged**